src/main/kotlin/no/nav/familie/ef/mottak/service/SøknadServiceImpl.kt

```diff
@@ -65,6 +69,25 @@ class SøknadServiceImpl(private val søknadRepository: SøknadRepository,
         }
     }
 
+    private fun erAktuellForFørsteSak(søknad: SøknadOvergangsstønad): Boolean {
+        val erAktuellForFørsteSakFT = featureToggleService.isEnabled("familie.ef.mottak.er-aktuell-for-forste-sak")
+        logger.info("Skal sjekke om søknad er aktuell for å plukkes som første sak i ny løsning: $erAktuellForFørsteSakFT")
+
+        if (!erAktuellForFørsteSakFT) {
+            return false
+        }
+        val dagensDato = LocalDate.now()
+
+
+        return søknad.barn.verdi.any {
```

Comment:
Ønsker vi å få inn søknader med 1 barn over 6 mnd, og 1 barn under 6 mnd?
Eller kun søknader med 1 barn under 6 mnd?

Reply:
Tror det holder at et av barna er under. Kan dobbeltsjekke med Mirja.

Reply:
Har sjekket - vi kjører på: et barn under, selv om noen er over holder. Vi sjekker jo heller ikke om dette nye barnet faktisk bor hos søker, men vi flytter bare disse sakene bort fra Mirjas oppgavebenk hvis det er dårlige kandidater for første sak.